components/DonationV1/index.ts

```diff
@@ -0,0 +1,3 @@
+import DonationV1 from './donationV1';
```

Comment:
I guess we have to discuss file naming convension with the team. It's pretty unusual (as for me) to see file naming **dontionV1.tsx** instead of **donation-v1.tsx**.

Reply:
Changed to donation-v1.tsx